srcjs/shinyapp.js

```diff
@@ -730,7 +730,7 @@ var ShinyApp = function() {
       prevTabIds.push($(this).find('> a').attr('href').replace(leadingHref,''));
     });
     prevTabIds = prevTabIds.map(Number);
-    var tabId = Math.max.apply(Math, prevTabIds) + 1;
+    var tabId = Math.max.apply(null, prevTabIds) + 1;
```

Comment:
I thought so too (see https://stackoverflow.com/a/1379556/6174455), and it certainly does the same thing. However, looking at the docs (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/max), you see they indeed use `null` instead of `Math` (code chunk from the second example):

```js
function getMaxOfArray(numArray) {
  return Math.max.apply(null, numArray);
}
```